docs/DevGuide/CodeReviewGuide.md

```diff
@@ -43,6 +43,9 @@ the class by the following line, which contains exactly 64 equal signs
   (<code>// \@{</code> and <code>// \@}</code>).
 * Use the [alternative tokens](http://en.cppreference.com/w/cpp/language/operator_alternative) `or`, `and` and `not` instead of `||`, `&&`, and `!`.
 * Use C-style Doxygen comments (`/*! ... */`) when using multi-line math, otherwise C-style and C++ style comments are accepted.
+* When addressing requests on a PR, the commit message must start with
+  `fixup:` followed by the a message describing the commit to later squash
```

Comment:
I would simplify this and just say "followed by a descriptive message" or "followed by the usual commit message".

(And if you disagree, at least fix "by the a message")